Fable.Remoting.MsgPack/Write.fs

```diff
@@ -285,19 +308,23 @@ let inline writeUnion union (out: Stream) (caseSerializers: Action<'a, Stream>[]
     let tag = tagReader union
     let fieldSerializers = caseSerializers.[tag]
 
-    out.WriteByte (Format.fixarr 2uy)
-    out.WriteByte (Format.fixposnum tag)
-
-    // save 1 byte if the union case has a single parameter
-    if fieldSerializers.Length <> 1 then
-        // todo one byte less for no args too (change fixarr above)
-        writeArrayHeader fieldSerializers.Length out
+    if fieldSerializers.Length = 0 then
+        out.WriteByte (Format.fixarr 1uy)
+        out.WriteByte (Format.fixposnum tag)
+    else
+        out.WriteByte (Format.fixarr 2uy)
+        out.WriteByte (Format.fixposnum tag)
```

Comment:
This will break the client if it's not updated at the same time as the server. Not sure if it deserves a major version bump or not (and not even that is going to prevent that scenario).

Reply:
We assume that every time someone is updating remoting packages, that they are updating both client and server to latest, so it's fine 🤞